recipe/meta.yaml

```diff
@@ -185,7 +185,7 @@ outputs:
         - pytables >=3.9.2
         - python >=3.11,<3.12
         - python-confluent-kafka >=1.9.2,<2
-        - pytorch >=2.1.0
+        - pytorch >=2.1.0,!=2.3.1                     # [aarch64]
```

Comment:
Unfortunately, the `# [aarch64]` makes this *only* apply to that architecture, so you also need an additional line:
```
- pytorch >=2.1.0   # [not aarch64]
```
And I think the official syntax is exactly two spaces between the end of the condition and the selector comment.